tests/TestCase/Http/Client/Auth/OauthTest.php

```diff
@@ -583,12 +541,16 @@ public function testRsaSigningStringWithPassphraseFile(): void
         $request = $auth->authentication($request, $options);
 
         $result = $request->getHeaderLine('Authorization');
-        $expected = 'jvTp/wX1TYtByB1m+Pbyo0lnCOLIsyGCH7wke8AUs3BpnwZJtAuEJkvQL2/9n4s5wUmUl4aCI4BwpraNx4RtEXMe5qg5T1LVTGliMRpKasKsW//e+RinhejgCuzoH26dyF8iY2ZZ/5D1ilgeijhV/vBka5twt399mXwaYdCwFYE=';
-        $this->assertStringContainsString(
-            'oauth_signature="' . $expected . '"',
-            urldecode($result)
-        );
+        $this->assertSignatureFormat($result);
         $expected = 0;
         $this->assertSame($expected, ftell($passphrase));
     }
+
+    protected function assertSignatureFormat($result)
+    {
+        $this->assertMatchesRegularExpression(
+            '/oauth_signature="[a-zA-Z0-9\/=+]+"/',
```

Comment:
I don't think there is much value in asserting the precise signed value. Knowing that there is a signed value is good enough.